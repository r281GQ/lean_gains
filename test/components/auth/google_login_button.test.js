import React from 'react';
import { shallow } from 'enzyme';
import GoogleLoginButton from './../../../src/components/auth/google_login_button';

describe('GoogleLoginButton test', () => {
  it('should render', () => {
    const component = shallow(<GoogleLoginButton />);
    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});
