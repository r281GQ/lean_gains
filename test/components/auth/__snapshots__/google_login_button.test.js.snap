// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleLoginButton test should render 1`] = `
<RaisedButton
  disabled={false}
  fullWidth={false}
  href="/api/auth/google"
  icon={
    <FontIcon
      className="fa fa-google"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    />
  }
  label="Log in with Google"
  labelPosition="before"
  primary={false}
  secondary={false}
/>
`;
